src/status_im/utils/gfycat/core.cljs

```diff
@@ -11,9 +11,12 @@
 
 (defn generate-gfy
   ([public-key]
-  (let [gen (rnd/rand-gen public-key)
-        first-adjective (pick-random gen adjectives/data)
-        second-adjective (pick-random gen adjectives/data)
-        animal (pick-random gen animals/data)]
-    (str first-adjective " " second-adjective " " animal)))
+  (case public-key
+    nil "Unknown"
+    "0" "Unknown"
+    (let [gen (rnd/rand-gen public-key)
+          first-adjective (pick-random gen adjectives/data)
+          second-adjective (pick-random gen adjectives/data)
+          animal (pick-random gen animals/data)]
+      (str first-adjective " " second-adjective " " animal))))
```

Comment:
Can you make the values in let all aligned?

Reply:
I've extracted this part into a `build-gfy` method. I think it make `generate-gfy` more readable. Though, for the alignement, I'm afraid I don't understand your comment. the values `gen` `first-adjective` `second-adjective` and `animal` have the same vertical alignement. This is first time I write Clojure code, so maybe I'm missing some common knowledge about this :)

Can you please explain more what you mean by `alignement` ?
Thanks!

Reply:
Yeah, I meant like this: